release_tester/test_driver.py

```diff
@@ -71,6 +71,11 @@ class TestDriver:
     def __init__(self, **kwargs):
         self.sitecfg = SiteConfig("")
         self.use_monitoring = kwargs["monitoring"]
+        tarball_limit = int(kwargs["tarball_count_limit"])
+        if tarball_limit == -1:
+            tarball_limit = 999999999
+        reporting.reporting_utils.TARBALL_LIMIT = tarball_limit
+        reporting.reporting_utils.TARBALL_COUNT = 0
```

Comment:
This will not work. The test-driver is instanciated several times by the full-* runners.